### ISSUE TITLE
❓ Question - when/how many times/how much time does it take ... to get synced 😉

### ISSUE THREAD

**Issue by Adam-it** [pnp/m365-sample-gallery#41]
## ❓ Question
Could you explain when (at what time?, how much more or less it takes?, how many times?) the sync process takes place ? 😉 
We could include this info in some wiki or readme 👍
I am asking at not so long ago we added CLI for M365 repo to be synced and I am wondering when should we be able to see sample coming based on this sample.json file -> https://github.com/pnp/cli-microsoft365/blob/main/docs/docs/sample-scripts/todo/cleanup-completed-todos/assets/sample.json

thanks for any kind of response 👍. Have a amazing week 🤩

**Comment by Adam-it** [comment]
hi again. 👋 
did someone had some time to revisit this one? currently after so long I think it is a bug 😟 and it seems that our script samples from the CLI M365 repo https://github.com/pnp/cli-microsoft365 are not getting synced 😱. 

As of now we have only one sample.json file to syn only single sample for tests reasons to check the process and open guidance how to refactor samples for contributors.  https://github.com/pnp/cli-microsoft365/blob/main/docs/docs/sample-scripts/todo/cleanup-completed-todos/assets/sample.json
But it seems this one is not getting synced 😟. Any ideas? tips? comments? 
🙏🙏🙏🙏🙏
@VesaJuvonen any help 😊

thanks for any kind of response 👍. Have a amazing week 🤩

**Comment by PaoloPia** [comment]
Hi! The sample is now available here: https://adoption.microsoft.com/en-us/sample-solution-gallery/sample/pnp-cli-microsoft365-cleanup-completed-todos/ . In order to make it work, I had to slightly fix the sample.json file. In particular:

- I changed the _name_ to be compliant with the common rules we have. Now it follows the format {organization}-{repo}-{sample} so it is: pnp-cli-microsoft365-cleanup-completed-todos
- I fixed the _updateDateTime_ attribute which was wrong, with a space at the end and that is not allowed
- I fixed the _longDescription_ attribute, which should be an array of strings and not a single string
- I created the "Microsoft To Do" product in the products.json system file (https://github.com/pnp/m365-sample-gallery/blob/main/system/products.json)

I will close this item, as it has been solved. Thanks!
